routes/place.routes.js

```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+const Place = require("../models/Place.model");
+
+// RUTA PARA MOSTRAR LA LISTA DE PLACES
+//URL con "/"
+router.get("/", (req, res, next) => {
+  Place.find()
+    //nombre de archivo
+    .then((allPlaces) => res.render("places-list", { places: allPlaces }))
+    .catch((err) => console.log(err));
+});
```

Comment:
- Aquí, cuando haces el `res.render`, le dices que renderice la vista `places-list` y le pasas el objeto, pero en este caso le estás pasando los sitios `allPlaces` dentro de la propiedad `places`, pero luego en la vista llamas a `allPlaces`.
- Para que te aparezcan, el código debería ser:

```suggestion
res.render("places-list", { allPlaces })
```